### PR TITLE
5605: Link up organisation page to edit user page

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/organisation/contextCreator.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const getContext = ({ organisation }) => ({
   users: organisation && organisation.users ? organisation.users.map(row => [
     {
       data: `${(`${row.firstName ? row.firstName : ''} ${row.lastName ? row.lastName : ''}`).trim()}` || '',
-      href: '#' || '#',
+      href: `/organisations/${organisation.organisationId}/${row.userId}` || '#',
```
Comment (alibombaye) [inline app/pages/organisation/contextCreator.js]: do we need the `|| '#'`

File: app/routes.js [hunk 2]
```diff
@@ -82,6 +82,12 @@ export const routes = (authProvider) => {
     return res.render('pages/adduser/template', context);
   }));
 
+  router.get('/organisations/:organisationId/:userId', authProvider.authorise(), withCatch(authProvider, async (req, res) => {
+    const { organisationId, userId } = req.params;
+    logger.info(`navigating to organisation: ${organisationId} edit user: ${userId} page`);
+    res.send('edit user page');
```
Comment (alibombaye) [inline app/routes.js]: I know it's just temp. But it will be the view user page right?

File: package.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
     "start": "cross-env NODE_ENV=production npm run build && cross-env NODE_ENV=production babel-node ./app/server",
     "test:ui": "cross-env NODE_ENV=test LOGGER_LEVEL=off babel-node testcafeRunner.js",
     "test:integration": "cross-env NODE_ENV=pipeline LOGGER_LEVEL=off babel-node testcafeRunner.js",
-    "test": "jest --coverage",
+    "test": "jest",
```
Comment (alibombaye) [inline package.json]: 😄

File: app/routes.js (hunk 2 above)
Comment (kat1906) [reply app/routes.js]: yep but I'm not changing it here - I have already started the next ticket!

**Pull request merged**